routes/todo.go

```diff
@@ -0,0 +1,63 @@
+package routes
+
+import (
+	"errors"
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"strconv"
+)
+
+func createErrorMap(err error) fiber.Map {
+	return fiber.Map{
+		"success": false,
+		"error":   err.Error(),
+	}
+}
+func (h handler) GetAll(c *fiber.Ctx) error {
+	todoList := h.DB.FetchAll()
+
+	return c.Status(fiber.StatusOK).JSON(fiber.Map{
+		"success": true,
+		"data":    todoList,
+	})
+
+}
+
+func (h handler) GetTodo(c *fiber.Ctx) error {
+	idParam := c.Params("id")
+	id, errConvert := strconv.Atoi(idParam)
+	if errConvert != nil {
+		err := fmt.Errorf("ID: %v is not from a valid type", idParam)
```

Comment:
Never start errors with capital letters, read the Go formatting guide and the tour of Go.

Reply:
I think your intent was to give the client information about its error, which is different from the compiler's error.
This is generally a good practice, but the actual error is lost, you need to log it with a wrap.

Reply:
> Never start errors with capital letters, read the Go formatting guide and the tour of Go.

fixed b01288b